src/error.rs

```diff
@@ -45,6 +46,39 @@ pub enum SnafuError {
   },
   #[snafu(display("Unrecognized representation `{}`", input))]
   UnrecognizedRepresentation { source: error::Error, input: String },
+  #[snafu(display("Unrecognized outgoing amount: `{}`", input))]
+  UnrecognizedAmount {
```

Comment:
I think I would call these `*Parse` instead of `Unrecognized*`.

Reply:
Thanks Casey - I'm pushing changes now that rename all of the `Unrecognized*` errors, except for `UnrecognizedRepresentation`, because that was already merged as part of #3192.